### ISSUE TITLE
Should no-duplicate-selectors complain about compound selectors?

### ISSUE THREAD

**Issue by davidtheclark** [stylelint/stylelint#807]
The rule complains about the following:

```cs
.btn {}

.link,
.btn {}
```

Because `.btn` is repeated. But that's not an objectionable pattern.

I'm wondering if the rule should only complain if the same *complete* selector recurs. So it would complain about the following:

```css
.btn {}
.btn {}
```

```css
.link, .btn {}
.link, .btn {}
```

But would not complain about this:

```css
.link {}
.btn {}
.link, .btn {}
```

Thoughts?

**Comment by jeddy3** [comment]
> Because `.btn` is repeated. But that's not an objectionable pattern.

Agreed.

> I'm wondering if the rule should only complain if the same complete selector recurs

Sgtm.
